Author: Mike Fitzpatrick <iraf@noao.edu>
Date: Mon Oct 21 16:03:18 2013 +0200
Subject: Use HSI environment when building F2C

Change appeared in version 2.16.1

--- a/unix/f2c/src/makefile.u
+++ b/unix/f2c/src/makefile.u
@@ -2,7 +2,8 @@
 
 .SUFFIXES: .c .o
 CC = cc
-CFLAGS = -O
+CFLAGS = -O -w $(HSI_CF)
+LDFLAGS = $(HSI_LF)
 SHELL = /bin/sh
 YACC = yacc
 YFLAGS =
@@ -24,7 +25,7 @@ MALLOC =
 
 OBJECTS = $(OBJECTSd) $(MALLOC)
 
-all: xsum.out f2c
+all: f2c
 
 f2c: $(OBJECTS)
 	$(CC) $(LDFLAGS) $(OBJECTS) -o f2c
