From 471d68815bf3f8fa4cf5b4353e1fb6041d90db71 Mon Sep 17 00:00:00 2001
From: Ole Streicher <olebole@debian.org>
Date: Sat, 18 Mar 2023 17:28:11 +0100
Subject: Use Makefiles in unix/f2c

---
 unix/f2c/libf2c/makefile.u | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/unix/f2c/libf2c/makefile.u b/unix/f2c/libf2c/makefile.u
index ccafa9df7..2f02a1ec3 100644
--- a/unix/f2c/libf2c/makefile.u
+++ b/unix/f2c/libf2c/makefile.u
@@ -120,6 +120,7 @@ sysdep1.h: sysdep1.h0
 # uncomment the following two lines:
 #main.o: main.c
 #	$(CC) -c -Donexit=on_exit -DSkip_f2c_Undefs main.c
+main.o: main.c signal1.h
 
 install: libf2c.a
 	cp libf2c.a $(LIBDIR)
@@ -128,13 +129,13 @@ install: libf2c.a
 clean:
 	rm -f libf2c.a *.o arith.h signal1.h sysdep1.h
 
-backspac.o:	fio.h
+backspac.o:	fio.h sysdep1.h
 close.o:	fio.h
 dfe.o:		fio.h
 dfe.o:		fmt.h
 due.o:		fio.h
 endfile.o:	fio.h rawio.h
-err.o:		fio.h rawio.h
+err.o:		fio.h rawio.h sysdep1.h
 fmt.o:		fio.h
 fmt.o:		fmt.h
 iio.o:		fio.h
@@ -165,7 +166,7 @@ signbit.o:	arith.h
 sue.o:		fio.h
 uio.o:		fio.h
 uninit.o:	arith.h
-util.o:		fio.h
+util.o:		fio.h sysdep1.h
 wref.o:		fio.h
 wref.o:		fmt.h
 wref.o:		fp.h
-- 
2.39.2

