From e2ce83e629c4cabc53e91a69a88d3cb358a2f302 Mon Sep 17 00:00:00 2001
From: Ole Streicher <olebole@debian.org>
Date: Mon, 22 Jan 2024 08:08:29 +0100
Subject: [PATCH] Upstream fix to make logical work

---
 unix/f2c/src/main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/unix/f2c/src/main.c b/unix/f2c/src/main.c
index 14aa10d8e..dd07f116c 100644
--- a/unix/f2c/src/main.c
+++ b/unix/f2c/src/main.c
@@ -262,7 +262,6 @@ set_externs(Void)
 	typesize[TYCLLIST] = 24;
 	typesize[TYALIST] = 16;
 	typesize[TYINLIST] = 208;
-	tyint = tylogical = tylog = TYLONG;
 	}
     else if (chars_per_wd > 0) {
 	typesize[TYADDR] = typesize[TYLONG] = typesize[TYREAL] =
-- 
2.43.0

